feconf.py

```diff
@@ -414,6 +414,11 @@ def get_empty_ratings():
 REDISHOST = 'localhost'
 REDISPORT = 6379
 
+# Replace project id with the correct project id when switching to the prod
+# server.
+OPPIA_PROJECT_ID = 'my-project-id'
+GOOGLE_APP_ENGINE_REGION = 'us-central1'
```

Comment:
Can we pick better defaults for the dev environment? E.g. maybe 'dev-project-id' and 'no-where1' or something?

Also, do we need to update the release process to include populating these values? How do we populate them correctly?

/cc @seanlip

Reply:
Oh, this is interesting.

Let's keep us-central1, that one is OK. But we do have to swap out the project ID during the release -- /cc @ankita240796. I think this might be the cause of an issue I ran into while deploying.